.pyup.yml

```diff
@@ -1,2 +1,5 @@
 label_prs: "enhancement"
-
+requirements:
+  - constraints.txt
```

Comment:
@vlall this should get pyup to look at constraints.txt, though I don't think we'll know for sure until it's merged.

Reply:
@alecpm Hmmm after merging the `constraints.txt` PR i’m seeing a waterfall of pyup PR’s since it’s looking to update `requirements.txt` back to include version pinning. We can discuss on Monday.